ccore/include/pyclustering/parallel/parallel.hpp

```diff
@@ -25,16 +25,16 @@
 #pragma once
 
 
+#include <algorithm>
 #include <cstddef>
 #include <functional>
 #include <future>
 #include <vector>
 
-#include <pyclustering/parallel/spinlock.hpp>
-
 
 #if defined(WIN32) || (_WIN32) || (_WIN64)
-#define PARALLEL_IMPLEMENTATION_PPL
+/* #define PARALLEL_IMPLEMENTATION_PPL */

```

Comment:
- [ ] Just remove it with a comment that `PARALLEL_IMPLEMENTATION_ASYNC_POOL` is more efficient.